model/user/user.go

```diff
@@ -454,3 +455,12 @@ func IsValidSubscriptionPreference(in string) bool {
 		return false
 	}
 }
+
+func GetSlackMemberId(username string) (string, error) {
+	u, err := FindBySlackUsername(username)
+	if err != nil {
+		return "", errors.Wrapf(err, "finding user by slack username '%s'", username)
+	}
+
```

Comment:
should we nil check for u here so we dont get a nil dereference

Reply:
done